frontend/www/js/omegaup/components/course/CourseCard.vue

```diff
@@ -5,7 +5,12 @@
       <div class="float-right" v-if="finishTime">{{ dueDate }}</div>
     </div>
     <div class="m-3">
-      <div class="float-left align-middle">{{ schoolName }}</div>
+      <div class="float-left align-middle">
+        <p>
+          {{ T.courseCardImpartedBy }}
+          <span class="font-weight-bold">{{ schoolName }}</span>
```

Comment:
se puede usar una cadena de traducción?

```suggestion
          {{ ui.formatString(T.courseCardImpartedBy, {school_name: ui.escape(schoolName)}) }}
```

donde `T.courseCardImpartedBy` sería `"Imparted by <span class="font-weight-bold">$(school_name)</span>"`

Reply:
Listo